test/SlowTests/Issues/RDBS-68.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using Xunit.Abstractions;
+using Xunit;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Utils;
+using Raven.Client;
+using Raven.Client.Documents.Operations.Expiration;
+using Raven.Client.Util;
+using Raven.Server.Documents.Expiration;
+using Raven.Server.ServerWide.Context;
+using Sparrow;
+
+namespace SlowTests.Issues
+{
+    public class RDBS_68 : RavenTestBase
+    {
+        public RDBS_68(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task CanDeleteAllExpiredDocsAfterRunningTheExpirationBundle()
+        {
+            using (var store = GetDocumentStore())
+            {
+                var config = new ExpirationConfiguration
+                {
+                    Disabled = false,
+                    DeleteFrequencyInSec = 1000
+                };
+
+                await ExpirationHelper.SetupExpiration(store, Server.ServerStore, config);
+
+                var expiry = SystemTime.UtcNow.AddMinutes(-5);
+
+                var deleteList = new List<string>();
+
+                var totalInBatch = ExpiredDocumentsCleaner.BatchSize;
```

Comment:
Either use smaller batch size or move this to stress

Reply:
that's 4096 documents in total

Reply:
Fixed.